documentation/src/docs/metrics.md

```diff
@@ -58,15 +97,17 @@ public interface OpStatsLogger {
 }
 ```
 
-- reportSuccessEvent() : Used to track Timer of a successful operation and will record the latency in Nanoseconds in required metric. 
-- reportFailEvent() : Used to track Timer of a failed operation and will record the latency in Nanoseconds in required metric.  
-- reportSuccessValue() : Used to track Histogram of a success value.
-- reportFailValue() : Used to track Histogram of a failed value. 
-- toOpStatsData() :  Used to support JMX exports and inner test.
-- clear : Used to clear stats for this operation.
+- `reportSuccessEvent()`: It is used to track the [Timer](http://metrics.dropwizard.io/3.1.0/manual/core/#timers) of a successful operation and will record the latency in nanoseconds in the required metric.
+- `reportFailEvent()`: It is used to track the [Timer](http://metrics.dropwizard.io/3.1.0/manual/core/#timers) of a failed operation and will record the latency in nanoseconds in required metric.  
+- `reportSuccessValue()`: It is used to track the [Histogram](http://metrics.dropwizard.io/3.1.0/manual/core/#histograms) of a success value.
+- `reportFailValue()`: It is used to track the [Histogram](http://metrics.dropwizard.io/3.1.0/manual/core/#histograms) of a failed value.
+- `toOpStatsData()`:  It is used to support the [JMX](https://metrics.dropwizard.io/3.1.0/manual/core/#jmx) exports and inner tests.
+- `clear`: It is used to clear the stats for this operation.
```

Comment:
@kevinhan88 `toOpStatsData()`:  It is used to support the [JMX](https://metrics.dropwizard.io/3.1.0/manual/core/#jmx) exports and inner tests.
In this para is JMX exports or reports.??

Reply:
I think what it means is that "toOpStatsData" could be used when JMX Reporter is switched on. However it is strongly recommended to turn on JMX Reporter in testing environment only.

Reply:
@kevinhan88 is the term `JMX exports` or JMX Reports`?Which one i should use

Reply:
In Codahale's terms it seems like "Reporter exports metrics from registries". So JMXReporter should be used, and I also suggest changing the "inner tests" to "unit tests".

Reply:
Accepted suggestion